cluster-autoscaler/FAQ.md

```diff
@@ -544,6 +559,34 @@ For example:
 
 For a complete list of the feature gates and their default values per Kubernetes versions, refer to the [Feature Gates documentation](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/).
 
+### How can I use ProvisioningRequest to run batch workloads
+
+ProvisioningRequest (or ProvReq) is a new namespaced Custom Resource that allows you to ask CA for capacity for groups of pods. For a detailed explanation of the ProvisioningRequest API, please refer to the
+[proposal](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/proposals/provisioning-request.md).
+
+#### Enabling ProvisioningRequest Support
+
+1. **Cluster Autoscaler Version**: Ensure you are using Cluster Autoscaler version 1.30.1 or later.
+
+2. **Feature Flag**: Enable ProvisioningRequest support by setting the following flag in your Cluster Autoscaler configuration:
+`--enable-provisioning-reques=true`.
+
+#### Supported ProvisioningClasses
+
+ClusterAutoscaler supports the following ProvisioningClasses:
+
+* `check-capacity.autoscaling.x-k8s.io`. 
+When using this class, Cluster Autoscaler performs following actions:
+
+  * **Capacity Check**: Determines if sufficient capacity exists in the cluster to fulfill the ProvisioningRequest.
+
+  * **Reservation** (if capacity is available): Reserves this capacity for the ProvisioningRequest for 10 minutes, preventing other ProvReqs from using it.
```

Comment:
Should we stress it out more that this is just 'reserving' in the sense of changing outcome of other check-capacity requests? It's not reserving it from other pods that may be created.
You say that at the end of the sentence, but I think it's worth highlighting. To me a default interpretation of reserving capacity would be a guarantee that no-one else will use it - but that's only true if everyone is blocking their job creation on check-capacity requests (i.e. in practice this only works for Kueue and no-one else).

Reply:
Thanks, added stress that we reserve capacity from other ProvReqs

Reply:
Just for my understanding (it doesn't have to go in the documentation):
It also blocks individual Pods that cluster-autoscaler processes from using that capacity. But because those pods would likely just be scheduled by kube-scheduler, in practice nothing prevents them from taking it.
Is that understanding correct?

Reply:
That's not true, at list not how it's implemented in #6880 right now - you're only booking capacity _after_ CA assigns all the pending pods to the nodes (which happens in FilterOutSchedulable / PodListProcessor) and you're only booking it until the end of CA loop. Meaning that any pending pod that shows up next loop will be allocated to this capacity before you re-book it.
As-is the implementation is only really preventing scale-down. To reserve from pods as you described the booking logic would need to happen before FilterOutSchedulable, not after it.

Reply:
You're right, thanks for correcting me! 
Aldo, your understanding is correct, it doesn't make a lot of sence to block CA from using the capacity, since CA just tries to predict behaviour of scheduler.